net/geoipupdate/Makefile

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=geoipupdate
+PKG_VERSION:=4.8.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/maxmind/geoipupdate/archive/refs/tags/
+PKG_HASH:=ca718c3ffcc595ef441363699888d20150f1d3a6583ac2d60bcbd34f052db09f
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_LICENSE:=Apache-2.0, MIT
+PKG_LICENSE_FILES:=LICENSE-APACHE, LICENSE-MIT
+PKG_MAINTAINER:=Matthew Hagan <mnhagan88@gmail.com>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/maxmind/geoipupdate
+GO_PKG_LDFLAGS_X:= \
+	main.defaultConfigFile=/etc/GeoIP.conf \
+	main.defaultDatabaseDirectory=/tmp/GeoIP
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/geoipupdate/Default
+  TITLE:=GeoIP Update
+  URL:=https://github.com/maxmind/geoipupdate
+  DEPENDS:=$(GO_ARCH_DEPENDS)
+endef
+
+define Package/geoipupdate
+$(call Package/geoipupdate/Default)
+  SECTION:=net
+  CATEGORY:=Network
+endef
+
+define Package/geoipupdate/Default/description
+The GeoIP Update program performs automatic updates of GeoIP2 and GeoIP
+Legacy binary databases. CSV databases are not supported.
+endef
+
+define Package/geoipupdate/description
+$(call Package/geoipupdate/Default/description)
+
+This package contains the program and config file.
+endef
+
+define Package/geoipupdate/install
+	$(call GoPackage/Package/Install/Bin,$(PKG_INSTALL_DIR))
+	
+	$(INSTALL_DIR) \
+	$(1)/usr/sbin \
+	$(1)/etc/init.d \
+	$(1)/lib/upgrade/keep.d
+
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/sbin/
+	$(INSTALL_CONF) $(PKG_BUILD_DIR)/conf/GeoIP.conf.default $(1)/etc/GeoIP.conf
+	$(INSTALL_BIN) ./files/geoipupdate.init $(1)/etc/init.d/geoipupdate
+	$(INSTALL_DATA) ./files/geoipupdate.upgrade $(1)/lib/upgrade/keep.d/geoipupdate
```

Comment:
Why this is required?

Reply:
Primarily we need to maintain the user's API key across upgrades in `/etc/GeoIP.conf`. Is this what you are referring to?

Reply:
Yes, but since the file is marked as conffile. This should not be necessary AFAIK.

Reply:
That should be it now. Confirmed `/etc/GeoIP.conf` is retained across sysupgrades with `geoipupdate.upgrade` removed.